libs/application/templates/children-residence-change/src/fields/Confirmation/index.tsx

```diff
@@ -1,27 +1,78 @@
 import React from 'react'
 import { useIntl } from 'react-intl'
-import { Box } from '@island.is/island-ui/core'
+import { Box, Text } from '@island.is/island-ui/core'
 import { confirmation, copyUrl } from '../../lib/messages'
-import { DescriptionText, CopyUrl } from '../components'
+import { CopyUrl, DescriptionText } from '../components'
+import { CRCFieldBaseProps } from '../..'
+import ContractAccordionOverview from '../components/ContractOverviewAccordion/ContractOverviewAccordion'
 
-const Confirmation = () => {
+const Confirmation = ({ application }: CRCFieldBaseProps) => {
   const { formatMessage } = useIntl()
+  const { answers, externalData } = application
+
   return (
     <>
       <Box marginTop={3}>
         <DescriptionText
           text={confirmation.general.description}
-          format={{ applicationNumber: '12345' }}
+          format={{
+            emailParagraph: answers.counterParty.email
+              ? formatMessage(
+                  confirmation.general.description.paragraphs.email,
+                  { email: answers.counterParty.email },
+                )
+              : '',
+            phoneNumberParagraph: answers.counterParty.phoneNumber
+              ? formatMessage(
+                  confirmation.general.description.paragraphs.phoneNumber,
+                  { phoneNumber: answers.counterParty.phoneNumber },
+                )
+              : '',
+            count: [
+              answers.counterParty.email,
+              answers.counterParty.phoneNumber,
+            ].reduce((value, item) => {
+              return item ? value + 1 : value
+            }, 0),
```

Comment:
Just question, could we do this instead of reduce?
```
count: [
              answers.counterParty.email,
              answers.counterParty.phoneNumber,
            ].filter(Boolean).length,
```